### ISSUE TITLE
doNotSendActsAsWhitelist not works properly as expected

### ISSUE THREAD

**Issue by shikendon** [DiscordSRV/DiscordSRV#381]
https://github.com/DiscordSRV/DiscordSRV/blob/79dc7d6d17f1df41e64a7edd527705e3c8049868/src/main/java/github/scarsz/discordsrv/objects/log4j/ConsoleAppender.java#L115-L117

In whitelist case, it means the `line` need to contain **all** phrases of the array.

Example:
```java
Boolean doNotSendActsAsWhitelist = true;
String line = "shikendon lost connection: Timed out";
String[] phrases = new String[]{"Timed out", "Whatever"};

for (String phrase : phrases)
    if (line.contains(phrase) == !doNotSendActsAsWhitelist) return;
System.out.println(line); // Unreachable
```

Expected:
```java
// import java.util.Arrays;

Boolean doNotSendActsAsWhitelist = true;
String line = "shikendon lost connection: Timed out";
String[] phrases = new String[]{"Timed out", "Whatever"};

if (Arrays.stream(phrases).parallel().anyMatch(line::contains) == !doNotSendActsAsWhitelist) return;

System.out.println(line); // shikendon lost connection: Timed out
```

Message should be send when `line` contains any phrases of whitelist.

**Comment by Scarsz** [comment]
You can make a PR for this if you want but DiscordSRV v1 isn’t under active development anymore.

**Comment by Serega007RU** [comment]
Can you sample this code so that he gets the phrase from the config?
And then too lazy to wait when you correct this mistake

**Comment by Scarsz** [comment]
merged with v1.16.5
